src/components/dialogs/addTodo/AddTodoDialog.tsx

```diff
@@ -66,35 +77,53 @@ export const AddTodoDialog:React.FC<Props> = ({ open, closeDialog, createTodo })
     setAssignedTo('');
   }
 
+  const fill = () => {
+    setTitle(prevData.title);
+    setState(prevData.state);
+    setAssignedTo(prevData.user_id.toString());
+    setDeadline(prevData.deadline);
+    setDescription(prevData.description);
+  }
+
   const close = () => {
-    clear();
     closeDialog();
   }
 
   const confirm = () => {
     if (isDataValid()) {
-      const data = { title, deadline, assigned_to: parseInt(assignedTo), description };
-      clear();
-      createTodo(data);
+      if(!isEdit) {
```

Comment:
Maybe you just need to check if your data have `id` then it's edit case otherwise create a new one. Does it make sense?

Reply:
I can check is `prevData` exists and if yes, it is edit case. But for this purpose I use  `isEdit` boolean variab.